### PR TITLE
Flow bids feature

### PR DESCRIPTION
**Pull request by vitalbadjo**

### CODE REVIEW

File: packages/sdk/src/sdk-blockchains/flow/test/sell-item.ts [hunk 1]
```diff
@@ -15,7 +16,8 @@ export async function sellItem(sell: FlowSell, itemId: ItemId, priceDecimals: st
 			contract: testFlowToken,
 		},
 		itemId,
+		originFees: [{ account: toUnionAddress(`FLOW:${FLOW_TESTNET_ACCOUNT_2.address}`), value: 200 }],
```
Comment (ex1st0r) [inline packages/sdk/src/sdk-blockchains/flow/test/sell-item.ts:19]: you should use convertFlowUnionAddress

File: packages/sdk/src/sdk-blockchains/flow/buy.test.ts [hunk 2]
```diff
@@ -25,8 +27,9 @@ describe("Flow buy", () => {
 		const order = await retry(10, 4000, () => apis.order.getOrderById({ id: orderId }))
 		expect(order.take.value.toString()).toEqual("0.1")
 
+		const originFees = [{ account: toUnionAddress(`FLOW:${FLOW_TESTNET_ACCOUNT_2.address}`), value: 200 }]
```
Comment (ex1st0r) [inline packages/sdk/src/sdk-blockchains/flow/buy.test.ts:30]: you should use convertFlowUnionAddress

File: packages/sdk/src/sdk-blockchains/flow/test/create-test-bid.ts [hunk 3]
```diff
@@ -12,6 +13,7 @@ export async function createTestBid(bid: FlowBid, itemId: ItemId) {
 			"@type": "FLOW_FT",
 			contract: testFlowToken,
 		},
+		originFees: [{ account: toUnionAddress(`FLOW:${FLOW_TESTNET_ACCOUNT_2.address}`), value: 200 }],
```
Comment (ex1st0r) [inline packages/sdk/src/sdk-blockchains/flow/test/create-test-bid.ts:16]: you should use convertFlowUnionAddress

File: packages/sdk/src/sdk-blockchains/flow/test/sell-item.ts (hunk 1 above)
Comment (vitalbadjo) [reply packages/sdk/src/sdk-blockchains/flow/test/sell-item.ts]: fixed

File: packages/sdk/src/sdk-blockchains/flow/buy.test.ts (hunk 2 above)
Comment (vitalbadjo) [reply packages/sdk/src/sdk-blockchains/flow/buy.test.ts]: fixed

File: packages/sdk/src/sdk-blockchains/flow/test/create-test-bid.ts (hunk 3 above)
Comment (vitalbadjo) [reply packages/sdk/src/sdk-blockchains/flow/test/create-test-bid.ts]: fixed

**Review by ex1st0r — APPROVED** [review]

**Pull request merged**